wicket-examples/src/main/java/org/apache/wicket/examples/ajax/builtin/EffectsPage.html

```diff
@@ -1,5 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <html xmlns:wicket="http://wicket.apache.org">
+<head>
```

Comment:
Why EffectsPage.html is modified ?

Reply:
Hm... no idea. Reverting back. Thanks for spotting this.

Reply:
Fixed.